### PR TITLE
ALCOM 0.1.7 (new cask)

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by i0ntempest** [comment]
This software is still in development, all releases are pre-releases.

**Review by miccal — APPROVED** [review]

**Comment by i0ntempest** [comment]
Description is inaccurate, VRChat Package Manager is another software released by VRChat devs and supports Windows only.

**Comment by bevanjkay** [comment]
Feel free to update it with a more appropriate description @i0ntempest

**Comment by i0ntempest** [comment]
Well the description I wrote was accurate but @miccal changed it. Anyways I changed it back.

**Comment by miccal** [comment]
> Well the description I wrote was accurate but @miccal changed it. Anyways I changed it back.

I changed it based on the description on the `homepage`:

> Fast Open Source command line (vrc-get) and graphical (ALCOM) client of VRChat Package Manager.

while also updating the `livecheck` and adding it to the `github_prerelease_allowlist.json` to pass our CI checks.

(You're welcome, but the way :unamused:)

**Comment by bevanjkay** [comment]
@i0ntempest Miccal was making a suggestion because the current doesn't really meet the description stanza requirements. It should describe what the application does, not what it is an alternative to. 

https://docs.brew.sh/Cask-Cookbook#stanza-desc

**Comment by i0ntempest** [comment]
Okay... I'm not trying to say anyone did anything bad, just trying to point out inaccuracies.
About the stanza requirement, here's my rationale: I could put "Graphical interface of vrc-get" which is what it is but then the user would think "wth is vrc-get?" upon seeing it, even if he/she is a VRChat user. Meanwhile my description is immediately understandable to any VRChat users because they most likely know what VCC is.
So, either the current one or "Graphical interface of vrc-get, open source alternative to VRChat Package Manager". I'll put in whichever you think is better.

**Comment by i0ntempest** [comment]
I've changed the description. Anything still needs to be done?

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @i0ntempest

File: Casks/a/alcom.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask "alcom" do
+  version "0.1.7"
+  sha256 "3d1f3e9372ae0f3ea141eafbcf8242da6f074eb0c0f14b3b7f6c048cfffa8e44"
+
+  url "https://github.com/vrc-get/vrc-get/releases/download/gui-v#{version}/ALCOM-#{version}-universal.dmg"
+  name "ALCOM"
+  desc "Graphical frontend of vrc-get, open source alternative to VRChat Package Manager"
+  homepage "https://github.com/vrc-get/vrc-get"
```
Comment (anatawa12) [inline Casks/a/alcom.rb:8]: homepage should be https://vrc-get.anatawa12.com/alcom I think

**Pull request merged**